cmd/veneur-emit/main.go

```diff
@@ -129,6 +134,18 @@ func main() {
 		return
 	}
 
+	if *traceID, err = inferTraceIDInt(*traceID, envTraceID); err != nil {
+		logrus.WithError(err).
+			WithField("env_var", envTraceID).
+			WithField("ID", "trace_id").
```

Comment:
I assume you know about `WithFields` but will mention it jic! Since it's only two fields I think it's a wash, but it's a bit cleaner for > 2.

Reply:
Hm, I kiinda hate the initializer form for its argument? But yeah, will adhere to convention next time! (: